libs/components/lists/src/lib/modules/repeater/repeater-item-roles.type.ts

```diff
@@ -0,0 +1,5 @@
+export type SkyRepeaterItemRolesType = {
+  item?: 'listitem' | 'option' | 'row' | undefined;
```

Comment:
Including `undefined` in the union type is redundant when using the `?` character. 

If you want to keep the `undefined` type in the union, you could remove the `?` character, especially if you want the consumer of the type to deliberately set each property (even if that deliberate value is `undefined`).